lib/batchVersions.js

```diff
@@ -141,6 +146,14 @@ export async function get({id, tokenizerId}) {
     }
     query['batchVersion.tokenizerId'] = tokenizerId;
   }
+
+  const cacheKey =
+    query['batchVersion.tokenizerId'] || query['batchVersion.id'];
```

Comment:
Hmm, these are in separate namespaces, so can we prefix this? It's paranoid to do so, but it would be bad to hit the case where they overlap and make things blow up. Alternatively, we can have two separate caches. This also influences the above max number ... we should double it if we aren't going to split the caches up.